.pyproject_generation/pyproject_custom.toml

```diff
@@ -22,3 +22,20 @@ all = ["hexkit[test]"]
 
 [project.urls]
 Repository = "https://github.com/ghga-de/hexkit"
+
+[tool.tox]
+legacy_tox_ini = """
```

Comment:
I had hoped they would support toml natively in tox 4, but looks like we still need to do it hat way.

Reply:
Yeah, seems so